app/controllers/catalog_controller.rb

```diff
@@ -202,6 +202,38 @@ def expire_session
       a.params.dig(:f, :format_f)&.include?('Database & Article Index')
     }
 
+    # Some filters (e.g., subject_f) are capable of driving meaning correlations;
+    # others are not, and either generate spurious correlations, or at best pointlessly add extra
+    # overhead to Solr request.
+    # Keys below indicate filters that we should not attempt to use for purpose of cacluating
+    # correlations -- either entirely (nil value) or for an array of certain filter values
+    @@CORRELATION_BLACKLIST = {
```

Comment:
i suggest we use `blocklist` or 'ignorelist' here

Reply:
Nice; side-benefit: those two terms you suggested actually differentiate in a meaningful way two different senses in which the previous term has historically been used (ambiguously) :-).
I think what we have here is an `ignorelist` ... will update accordingly.